plugin/src/test/java/co/brainly/onesky/task/DownloadTranslationsTaskTest.kt

```diff
@@ -0,0 +1,77 @@
+package co.brainly.onesky.task
+
+import co.brainly.onesky.client.util.enqueueResponseWithJsonFile
+import co.brainly.onesky.util.buildFile
+import co.brainly.onesky.util.settingsFile
+import okhttp3.mockwebserver.MockWebServer
+import org.gradle.testkit.runner.GradleRunner
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+import org.junit.jupiter.api.Assertions.assertEquals as assertEquals
+
+class DownloadTranslationsTaskTest {
```

Comment:
loose thoughts - you can test that here - in what seems to me like end to end test, but really it's just test of `androidDirName` getter. So test on it would seem sufficient. BUT since you have it here it will make any refactor failure proof and this is something I like. Maybe there is some place little deeper - that would make it possible to test it without calling HTTP API, and at the same time being refactor-proof. 

To summarize - looks good, but maybe there is a better level for this test.

Reply:
Thank you, I really wanted to try out writing tests with GradleRunner, which is a recommended tool for writing Gradle plugins.
Will leave it for now as is, but this test should actually cover fully working example - I will treat this test as a good starting point for writing a test which fully covers responses from OneSky.